src/controllers/mainController.ts

```diff
@@ -143,7 +143,7 @@ export default class MainController implements vscode.Disposable {
         return new Promise<boolean>( (resolve, reject) => {
             // Ensure telemetry is disabled
             Telemetry.disable();
-            SqlToolsServerClient.instance.initialize(self._context).then(serverResult => {
+            SqlToolsServerClient.instance.initialize(self._context).then(async serverResult => {
```

Comment:
Why is this being made async? I'm not seeing any changes that would require that